### DISCUSSION TITLE
Setup on LXC?

### DISCUSSION THREAD

**Discussion by dominikheinz** [FreshRSS/FreshRSS#4435] [Q&A]
Hello,
I wonder how I can setup FreshRSS in an LXC container.
I am running a proxmox instance at home, and I feel like creating a separate VM just for FreshRSS is a bit overkill. 
Therefore, I was wondering if I can just setup FreshRSS in an LXC container, ideally without docker.
If that is possible, can someone provide some assistance how I could set this up?

**Comment by Alkarex** [comment]
We do not have instructions for LXC at the moment, but installing FreshRSS on any Linux system is [relatively easy](https://freshrss.github.io/FreshRSS/en/admins/03_Installation.html) and straightforward. It only requires PHP and a Web server such as Apache, and a database server is optional (defaults to built-in SQLite), so pretty standard.

You can get inspired by our Dockerfile [for Debian ](https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/Dockerfile) or [for Alpine](https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/Dockerfile-Alpine) for e.g. the precise packages to install.
